doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1167,6 +1177,11 @@ Controlling Typing Flags
    :cmd:`Print Assumptions`. It has the same effect as `-type-in-type` command line
    argument (see :ref:`command-line-options`).
 
+.. attr:: typing(universes)
```

Comment:
It should be referenced in the list of attributes supported by the relevant commands.